src/servicex_did_finder_lib/servicex_adaptor.py

```diff
@@ -72,7 +72,7 @@ def put_file_add(self, file_info):
             try:
                 mesg = {
                     "timestamp": datetime.now().isoformat(),
-                    "file_path": self._prefix_file(file_info['file_path']),
+                    "file_path": [self._prefix_file(fp) for fp in file_info['file_path']],
```

Comment:
Isn't the JSON property now `paths` instead of `file_path`?

Reply:
true. Will fix now